modules/apps/29-fee/ibc_middleware.go

```diff
@@ -219,7 +219,9 @@ func (im IBCMiddleware) OnRecvPacket(
 	packet channeltypes.Packet,
 	relayer sdk.AccAddress,
 ) exported.Acknowledgement {
+	logger := im.keeper.Logger(ctx)
 	if !im.keeper.IsFeeEnabled(ctx, packet.DestinationPort, packet.DestinationChannel) {
+		logger.Info("fee middleware module is disabled")
```

Comment:
I'm not sure how useful this is. This will be logged on every receive on non fee enabled chanenls. Seems unnecessary to me?